packages/jmix-front-generator/src/generators/sdk/all/options.ts

```diff
@@ -0,0 +1,3 @@
+import {CommonGenerationOptions} from "../../../common/cli-options";
+
+export type Options = CommonGenerationOptions;
```

Comment:
I think we don't need this file if we are just using `CommonGenerationOptions`, can use it directly in `index.ts`, it's more clear that we aren't using any specific options.

Reply:
@vyacheslav-pushkin
fixed